scripts/rabbitmq-server-ha.ocf

```diff
@@ -711,7 +721,9 @@ reset_mnesia() {
     if $make_amnesia ; then
         kill_rmq_and_remove_pid
         ocf_run rm -rf "${MNESIA_FILES}"
-        ocf_log warn "${LH} Mnesia files appear corrupted and have been removed from ${MNESIA_FILES}."
+        mnesia_schema_location="${OCF_RESKEY_mnesia_schema_base}/Mnesia.$(rabbit_node_name $(get_hostname))"
```

Comment:
I would prefer to keep mnesia_schema_location lowercase - it is just a local variable and we have them in lowercase everywhere. I'd rather make MNESIA_FILES a local variable later and make it lowercase as well.

Regarding files like MnesiaCore.rabbit@n2_1473_324370_784463, I don't think we should remove them. These are core dumps from Mnesia and they might help us to perform RCA in case Mnesia fails. In my case just removing schema directory helped to restore RabbitMQ cluster.